src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -197,7 +211,15 @@ void AddSceneParse(CommandHandlerInterface::HandlerContext & ctx, const CommandD
     auto fieldSetIter = req.extensionFieldSets.begin();
 
     uint8_t EFSCount = 0;
-    SceneData storageData(req.sceneName, transitionTimeMs);
+
+    uint32_t featureMap = 0x00;
+    ReturnOnFailure(AddResponseOnError(ctx, response, Attributes::FeatureMap::Get(ctx.mRequestPath.mEndpointId, &featureMap)));
```

Comment:
I think it would be a better approach to read the the featuremap and use it afterwords instead of merging both operations.

Reply:
We are not using the feature map here but the error generated by reading the featureMap attribute.